src/server/services/education/education.service.ts

```diff
@@ -11,6 +11,57 @@ const educationService = {
   getUniqueTypes: async () => {
     return educationRepository.findUniqueTypes();
   },
+  deleteById: async (id: string) => {
+    return educationRepository.deleteById(id);
+  },
+  create: async (
+    type: string,
+    title: string,
+    description: string,
+    precondition: string,
+    duration: string,
+    lecturers: string,
+    courseDuration: string,
+    renewalDuration: string,
+    topics: string,
+  ) => {
+    return educationRepository.create(
+      type,
+      title,
+      description,
+      precondition,
+      duration,
+      lecturers,
+      courseDuration,
+      renewalDuration,
+      topics,
+    );
+  },
+  update: async (
+    id: string,
+    type: string,
+    title: string,
+    description: string,
+    precondition: string,
+    duration: string,
+    lecturers: string,
+    courseDuration: string,
+    renewalDuration: string,
+    topics: string,
+  ) => {
+    return educationRepository.update(
+      id,
+      type,
+      title,
+      description,
+      precondition,
+      duration,
+      lecturers,
+      courseDuration,
+      renewalDuration,
+      topics,
+    );
+  },
```

Comment:
Same comment as for the create.

Reply:
`update` has an additional parameter `id` which  `create` doesn't.

Reply:
Than the `EducationFormData` should have `id?: string`